src/lib/db-listeners/db-supports-listener.ts

```diff
@@ -32,26 +37,54 @@ export class DBSupportsListener {
   private handleSupportsUpdate = (snapshot: firebase.database.DataSnapshot) => {
     const {supports} = this.db.stores;
     const dbSupports = snapshot.val();
+    let audienceType: AudienceEnum | undefined;
```

Comment:
Ah you can ignore this - I ended up with a better method of figuring out the audience type described in the PR summary